cli/running/instance_test.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 const (
-	instTestAppDir = "./test_app"
+	instTestAppDir  = "./test_app"
+	instTestDataDir = "./test_app"
```

Comment:
I think you can use some temporary directory for this purpose.

Reply:
Add a test for `DataDir`.
Remove `DataDir` in  `cleanupTestInstance`.

Reply:
agreed, added